randovania/games/prime1/json_data/Chozo Ruins.txt

```diff
@@ -2507,7 +2541,9 @@ Extra - aabb: [754.19855, -343.01227, 58.286064, 793.17523, -254.9616, 100.00249
           All of the following:
               Morph Ball
               Any of the following:
-                  Boost Ball and After Reflecting Pool Water Drained
+                  All of the following:
+                      After Reflecting Pool Water Drained
+                      Boost Ball or Movement (Advanced)
```

Comment:
Put the video as a comment here.

Reply:
Isn't this one a discord comment? Is that fine?

Reply:
Yeah that's fine